tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -810,3 +810,121 @@ def solve_tridiag(
     solve_tridiag(a, b, c, d, out=out, offset_provider={})
 
     np.allclose(expected, out)
+
+
+def test_ternary_operator(reduction_setup):
+    Edge = reduction_setup.Edge
+    num_edges = reduction_setup.num_edges
```

Comment:
No need to use the much heavier `reduction_setup` fixture. Just initialize these two directly here. Same below for the non-reduction test cases.